plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergFailureRecoveryTest.java

```diff
@@ -148,18 +148,88 @@ public void testDelete()
                 .finishesSuccessfully();
     }
 
+    // Copied from BaseDeltaFailureRecoveryTest
     @Override
     public void testUpdate()
     {
-        assertThatThrownBy(super::testUpdate)
-                .hasMessageContaining("This connector does not support updates");
-    }
+        // Test method is overriden because method from superclass assumes more complex plan for `UPDATE` query.
```

Comment:
cc @losipiuk @arhimondr for `BaseIcebergFailureRecoveryTest`

Reply:
Looks good to me